### PR TITLE
[merged] Bootstrap can now copy over the CA for etcd if provided.

### PR DESCRIPTION
**Pull request by ashcrow**
This is primarily for Flannel which will refuse to connect to etcd over
TLS if it doesn't have the proper CA available.

### CODE REVIEW

File: src/commissaire/data/ansible/playbooks/bootstrap.yaml [hunk 1]
```diff
@@ -24,6 +24,11 @@
         dest: "{{ commissaire_etcd_client_key_path }}"
         src: "{{ commissaire_etcd_client_key_path_local }}"
       when: "{{ commissaire_etcd_use_cert }}"
+    - name: Add Etcd CA
+      copy:
+        dest: "{{ commissaire_etcd_ca_path }}"
+        src: "{{ commissaire_etcd_ca_path_local }}"
+      when: "commissaire_etcd_ca_path is defined"
```
Comment (mbarnes) [inline src/commissaire/data/ansible/playbooks/bootstrap.yaml]: Could we use this style of conditional for all the certificate files in the playbook and drop the `use_cert` variables?  Seems a little cleaner to me, and the server already checks that both parts of the cert/key pair are present for etcd and kube.

Comment (ashcrow) [inline src/commissaire/data/ansible/playbooks/bootstrap.yaml]: Yup! No reason we can't move to it. I'll do a follow up to move others over after this PR is done.

File: src/commissaire/script.py [hunk 2]
```diff
@@ -163,12 +166,15 @@ def main():  # pragma: no cover
         'protocol': config.etcd['uri'].scheme,
     }
 
-    # We need both args to use a client side cert for etcd
+    if args.etcd_ca_path:
```
Comment (mbarnes) [inline src/commissaire/script.py]: Should this be `if bool(args.etcd_ca_path):` ?

Not quite sure why it's necessary but you cast to `bool` in other places.

Comment (ashcrow) [inline src/commissaire/script.py]: Good point. I don't think it's needed but it's best to be consistent.

**Comment by mbarnes** [comment]
@cgwalters-bot test

**Comment by mbarnes** [comment]
@cgwalters-bot try

**Comment by mbarnes** [comment]
@cgwalters-bot r+

**Pull request closed without merging**